process/nodeStatusProcessor_test.go

```diff
@@ -73,6 +74,58 @@ func TestNodeStatusProcessor_GetConfigMetrics(t *testing.T) {
 
 }
 
+func TestNodeStatusProcessor_GetTotalStakedErr(t *testing.T) {
```

Comment:
add this endpoint also in the `testing` package

Reply:
done.